break/viewABCD.md

```diff
@@ -0,0 +1,253 @@
+---
+jupyter:
+  jupytext:
+    formats: ipynb,md
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.2'
+      jupytext_version: 1.7.1
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+```python
+import pandas as pd
+import numpy as np
+import plotly.express as px
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+import os
+from pathlib import Path
+from glob import glob
+from collections import namedtuple
+
+import requests
+```
+
+### Gather all the data elements from all the txt files
+
+The ABCD3 folder contains a number of tab-delimited text files with ABCD data.
+We will collect all of these files and then load them into pandas DataFrames
+to compile and access the data.
+
+```python
+data_path = Path("/home/jovyan/ABCD3")
+files = sorted(data_path.glob("*.txt"))
+```
+
+```python
+# We store the info in 4 different Python datatypes
+data_elements = []
+data_structures = {}
+event_names = set()
+StructureInfo = namedtuple("StructureInfo", field_names=["description", "eventnames"])
+
+for fl in files:
```

Comment:
I know this is a minor thing, but given the goal of this notebook, I think we should use longer, but more informative, variable names. E.g., `text_file` instead of `fl`.